src/components/Modal/Modal.module.css

```diff
@@ -0,0 +1,82 @@
+.modalContainer{
+  position: fixed;
+  top:0;
+  left:0;
+  right:0;
+  bottom:0;
+  display: flex;
+  justify-content: center;
+  background-color: rgba(0, 0, 0, 0.7);
+  z-index: 999;
```

Comment:
Using such a big z-index: value is conviniently a bad practice. I was looking for articles about this and found [this one with a good argumentation](https://devrix.com/tutorial/css-best-practices-bad-css/).

Reply:
I am not really good at vanilla css 😢 . I was using tailwind for personal projects most of the time. So I checked their z-index convention and applied their rule now. basically from 0 to 50 with step 10. Thanks for the information.

Reply:
I still do not like but if you used any conventional approach -- it is good.
Resolved ✅